integration-tests/src/tests/ct_migration.rs

```diff
@@ -46,10 +37,8 @@ fn mock_hrmp_establishment(project_id: u32) {
 		assert_ok!(PolimecFunding::do_handle_channel_accepted(channel_accepted_message));
 	});
 
-	PenNet::execute_with(|| {
-		println!("penpal events:");
-		dbg!(PenNet::events());
-	});
+	// Required for passing migration ready check.
```

Comment:
Forgot to leave this comment. Sorry if it gave you a headache to finds this out :(